var/spack/repos/builtin/packages/mdspan/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Mdspan(CMakePackage):
+    """Extension of std::span to enable multidimensional arrays"""
+
+    homepage = "https://github.com/kokkos/mdspan/tree/stable"
+    url = "https://github.com/kokkos/mdspan/archive/refs/tags/mdspan-0.6.0.zip"
+    git = "https://github.com/kokkos/mdspan.git"
+
+    variant("examples", default=True, description="Enable examples")
+    variant("tests", default=False, description="Enable tests")
+    variant("benchmarks", default=False, description="Enable benchmarks")
+
+    depends_on("benchmark", when="+benchmarks")
+    depends_on("googletest@main", when="+tests")
+
+    version("stable", branch="stable")
+    version("0.6.0", sha256="d6b7b9d4f472106df1d28729bd8383a8a7ea7938adf9f82d3be9c151344830d9")
```

Comment:
```suggestion
    version("stable", branch="stable")
    version("0.6.0", sha256="d6b7b9d4f472106df1d28729bd8383a8a7ea7938adf9f82d3be9c151344830d9")
    
    variant("examples", default=True, description="Enable examples")
    variant("tests", default=False, description="Enable tests")
    variant("benchmarks", default=False, description="Enable benchmarks")

    depends_on("benchmark", when="+benchmarks")
    depends_on("googletest@main", when="+tests")
```

Small nitpick, but it makes it easier to review packages if they follow the common format of 
```
1. versions
2. variants
3. dependencies
```

Reply:
oh good call, Thanks!